backend/scheduler/tasks.py

```diff
@@ -57,13 +59,27 @@ def execute_pipeline_task(
     name: Any,
 ) -> None:
     logger.info(f"Executing pipeline name: {name}")
-
     try:
         logger.info(f"Executing workflow id: {workflow_id}")
         tenant: Organization = (
             get_tenant_model().objects.filter(schema_name=org_schema).first()
         )
         with tenant_context(tenant):
+            subscription_plugin_available = (
+                os.environ.get("SUBSCRIPTION_PLUGIN_AVAILABLE", "False") == "True"
```

Comment:
Can we replace the env variable with some constants or checking the load_plugins() instaed of saving it in env variable?

Reply:
Moved to plugin check